ncm-spma/src/main/perl/spma/ips.pm

```diff
@@ -364,7 +392,7 @@ sub run_pkg_command
     #   (4 for /usr/bin/pkg, 1 for /usr/bin/spma-run)
     #
     my ($self, $cmd, $log, $exit_void) = @_;
-    my $proc = CAF::Process->new($cmd, log => $self);
+    my $proc = CAF::Process->new($cmd, log => $self, keeps_state => 1);
```

Comment:
All 3 uses of `run_pkg_command()` in the module keep state.  When I wrote this in 2015 (the code has been in use for two years already I just omitted to push it upstream), I didn't envisage any need to execute a command that changed state and that remains the case today.

Would you prefer I exposed that as an argument to the method and set this argument to 1 for each invocation?